Library/ViewModels/RewardCellViewModel.swift

```diff
@@ -215,7 +217,7 @@ RewardCellViewModelOutputs {
 
     self.cardViewBackgroundColor = Signal.combineLatest(allGoneAndNotABacker, self.boundStylesProperty.signal)
       .map(first)
-      .map { $0 ? .ksr_grey_100 : .white }
+      .map { _ in .white }
```

Comment:
you can kill both `maps` and just use `mapConst(.white)`. Or, you may not even need this output now?